### PR TITLE
S02US10: modifiaction du style

### PR DESCRIPTION
**Pull request by maxime-guichard**

### CODE REVIEW

**Review by Wowlfy — DISMISSED** [review]

**Review by Wowlfy — APPROVED** [review]

File: templates/component/_footer.html.twig [hunk 1]
```diff
@@ -1,55 +1,66 @@
...
+                <!-- Section: Adresse -->
+				<p>1 faubourg Bannier</p>
+                <p>45000 Orléans</p>
+                <!-- Section: Adresse -->
+			</div>
+			<!--Grid column-->
+		</div>
 	</div>
-	<!-- Copyright -->
 </footer>
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:66]: wait, so you have 2 footers?

File: templates/base/_footer.html.twig [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+			<div class="col-lg-4 col-md-6 mb-4 mb-md-0 pt-2">
+                <!-- Section: Adresse -->
+				<p>1 faubourg Bannier</p>
+                <p>45000 Orléans</p>
+                <!-- Section: Adresse -->
+			</div>
+			<!--Grid column-->
+		</div>
+	</div>
+</footer>
```
Comment (LucyMerlier) [inline templates/base/_footer.html.twig:66]: wait, so you have 2 footers?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments + lighten up those 3 images a bit (should be <200kb, especially for small icons like that)

File: templates/base/_footer.html.twig (hunk 2 above)
Comment (maxime-guichard) [reply templates/base/_footer.html.twig:66]: c'est supprimé

**Review by Erdnaxela45 — DISMISSED** [review]

File: templates/component/_footer.html.twig [hunk 3]
```diff
@@ -1,55 +1,66 @@
...
-			<div class="col justify-content-start">
-				<a href="{{ path('company_quotation') }}" class="text-white text-decoration-none">Demander un devis</a>
-			</div>
-			<div class="col justify-content-end">
-				<a href="{{ path('contact') }}" class="text-white text-decoration-none">Nous contacter</a>
-			</div>
-		</div>
+	<!-- Grid container -->
+	<div
+		class="container pt-3">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:6]: why is this div's class attribute on the next line?

File: templates/component/_footer.html.twig [hunk 4]
```diff
@@ -1,55 +1,66 @@
...
 		<!--Grid row-->
-	</section>
-	<!-- Section: Links -->
-</div>
+		<div
+			class="row py-3">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:9]: why is this div's class attribute on the next line?

File: templates/component/_footer.html.twig [hunk 5]
```diff
@@ -1,55 +1,66 @@
...
+			<!--Grid column-->
+			<div
+				class="col-lg-4 col-md-6 mb-4 mb-md-0">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:12]: why is this div's class attribute on the next line?

File: templates/component/_footer.html.twig [hunk 6]
```diff
@@ -1,55 +1,66 @@
...
-	<!-- Section: Social media -->
-	<section
-		class="d-flex justify-content-around mb-4">
-		<!-- Facebook -->
-		<a class="m-1" href="https://www.facebook.com/studentinterim" role="button">
-			<img src="{{ asset('build/images/facebook.png') }}" alt="logo facebook">
-		</a>
+				<!-- Section: Social media -->
+				<section
+					class="d-flex justify-content-around">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:16]: why is this section's class attribute on the next line?

File: templates/component/_footer.html.twig [hunk 7]
```diff
@@ -1,55 +1,66 @@
...
+			<!--Grid column-->
+			<div class="col-lg-4 col-md-6 mb-4 mb-md-0 pt-2">
+				<!-- List: Links -->
+					<ul class="list-unstyled mb-0">
+						<li class="pb-3">
+							<a href="{{ path('company_quotation') }}" class="text-white text-decoration-none">Nous contacter</a>
+						</li>
+						<li>
+							<a href="{{ path('contact') }}" class="text-white text-decoration-none">Demander un devis</a>
+						</li>
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:50]: you switch those link's hrefs (your contact link goes to the quotation request page and your quotation request link goes to your contact page)

File: templates/component/_footer.html.twig [hunk 8]
```diff
@@ -1,55 +1,66 @@
...
-	<!-- Copyright -->
-	<div class="text-center" style="background-color: rgba(0, 0, 0, 0.2);">
-		<p class="text-white">© 2020 Copyright: Student-Interim</p>
+			<!--Grid column-->
+			<div class="col-lg-4 col-md-6 mb-4 mb-md-0 pt-2">
+                <!-- Section: Adresse -->
+				<p>1 faubourg Bannier</p>
+                <p>45000 Orléans</p>
+                <!-- Section: Adresse -->
+			</div>
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:62]: indentation

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: templates/component/_footer.html.twig (hunk 3 above)
Comment (maxime-guichard) [reply templates/component/_footer.html.twig:6]: it's not my fault i press ctrl+maj+i

File: templates/component/_footer.html.twig (hunk 4 above)
Comment (maxime-guichard) [reply templates/component/_footer.html.twig:9]: it's not my fault i press ctrl+maj+i

File: templates/component/_footer.html.twig (hunk 5 above)
Comment (maxime-guichard) [reply templates/component/_footer.html.twig:12]: it's not my fault i press ctrl+maj+i

File: templates/component/_footer.html.twig (hunk 6 above)
Comment (maxime-guichard) [reply templates/component/_footer.html.twig:16]: it's not my fault i press ctrl+maj+i

File: templates/component/_footer.html.twig (hunk 8 above)
Comment (maxime-guichard) [reply templates/component/_footer.html.twig:62]: it's not my fault i press ctrl+maj+i

**Review by endricklosange — APPROVED** [review]

File: templates/component/_navbar_admin.html.twig [hunk 9]
```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-sm bg-primary navbar-dark">
+gt branch*<nav class="navbar navbar-expand-sm bg-primary navbar-dark">
```
Comment (LucyMerlier) [inline templates/component/_navbar_admin.html.twig:1]: what is this "git branch" doing here?

File: templates/component/_footer.html.twig (hunk 3 above)
Comment (LucyMerlier) [reply templates/component/_footer.html.twig:6]: not my problem, fix it.

File: templates/component/_footer.html.twig (hunk 4 above)
Comment (LucyMerlier) [reply templates/component/_footer.html.twig:9]: not my problem, fix it.

File: templates/component/_footer.html.twig (hunk 5 above)
Comment (LucyMerlier) [reply templates/component/_footer.html.twig:12]: not my problem, fix it.

File: templates/component/_footer.html.twig (hunk 6 above)
Comment (LucyMerlier) [reply templates/component/_footer.html.twig:16]: not my problem, fix it.

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Erdnaxela45 — DISMISSED** [review]

File: templates/component/_footer.html.twig [hunk 10]
```diff
@@ -1,55 +1,63 @@
 <!-- Footer -->
 <footer
 	class="bg-secondary text-center text-white">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:3]: forgot one => class attribute of footer should be on same line

File: templates/component/_footer.html.twig [hunk 11]
```diff
@@ -1,55 +1,63 @@
...
+			</section>
+			<!-- List: Links -->
+		</div>
 
-	<!-- Copyright -->
-	<div class="text-center" style="background-color: rgba(0, 0, 0, 0.2);">
-		<p class="text-white">© 2020 Copyright: Student-Interim</p>
+		<!--Grid column-->
+		<div
+			class="col-lg-4 col-md-6 mb-4 mb-md-0 pt-2">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:55]: and another one => class attribute of div should be on same line

File: templates/component/_footer.html.twig [hunk 12]
```diff
@@ -1,55 +1,63 @@
...
+					<a class="m-1" href="https://youtube.com/channel/UCqQPYBaPQ7eRUtB-GKWIniQ" role="button">
+						<img src="{{ asset('build/images/youtube.png') }}" alt="logo youtube">
+					</a>
+				</section>
+			</div>
+			<!--Grid column-->
+
+			<!--Grid column-->
+			<div
+				class="col-lg-4 col-md-6 mb-4 mb-md-0 pt-2">
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:39]: and another one => class attribute of div should be on same line

File: templates/component/_footer.html.twig [hunk 13]
```diff
@@ -1,55 +1,63 @@
...
+			<!-- Section: Adresse -->
+			<p>1 faubourg Bannier</p>
+			<p>45000 Orléans</p>
+			<!-- Section: Adresse -->
+		</div>
+		<!--Grid column-->
 	</div>
-	<!-- Copyright -->
-</footer>
+</div></footer>
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:63]: why is this div closing tag on the same line as the footer closing tag? => check your number of divs, I suspect you have one too much div closing tag

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Erdnaxela45 — DISMISSED** [review]

File: templates/component/_footer.html.twig [hunk 14]
```diff
@@ -1,55 +1,60 @@
...
+				<!-- List: Links -->
+				<ul class="list-unstyled mb-0">
+					<li class="pb-3">
+						<a href="{{ path('contact') }}" class="text-white text-decoration-none">Nous contacter</a>
+					</li>
+					<li>
+						<a href="{{ path('company_quotation') }}" class="text-white text-decoration-none">Demander un devis</a>
+					</li>
+				</ul>
+			</section>
```
Comment (LucyMerlier) [inline templates/component/_footer.html.twig:47]: i guess you meant to close the div that you openned on line 37, not a section

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by endricklosange — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**